File: python/7 kyu/318.py
# https://www.codewars.com/kata/583989556754d6f4c700018e/train/python

# Passed

from math import gcd

def multiples(s1,s2,s3):
    result = []
    multiplier = 1
    base_val = s1 * s2 / gcd(s1, s2)
    base_val = int(base_val) if base_val.is_integer() else base_val
    val = base_val
    while val < s3:
        result.append(val)
        multiplier += 1
        val = base_val * multiplier
        
    return result

output = multiples(3,4,40)
print(output)

File: nodejs/7 kyu/318.js
// https://www.codewars.com/kata/51f2b4448cadf20ed0000386/train/javascript

// Passed

function removeUrlAnchor(url) {
  let result = url.split("#")[0];
  return result;
}

const output = removeUrlAnchor("www.codewars.com/katas/?page=1#about");
console.log(output);